From cd8c778456db68de1c75216fe36a7682abbb6aac Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Wed, 13 Dec 2017 21:01:01 +0800
Subject: [PATCH] net: rfkill: fixes the warning with gcc-7.2 build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

when used the arm-linux-gnueabihf-gcc-7.2.1, had the below
warning information:
net/rfkill/rfkill-bt.c:711:5: warning: this ‘if’ clause does not
guard... [-Wmisleading-indentation] error, forbidden warning

Change-Id: I7834078bb731e1115f6e8328a56158b0d9f2991e
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 net/rfkill/rfkill-bt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/rfkill/rfkill-bt.c b/net/rfkill/rfkill-bt.c
index 920eaa3981e2..cffbeec17909 100644
--- a/net/rfkill/rfkill-bt.c
+++ b/net/rfkill/rfkill-bt.c
@@ -709,7 +709,7 @@ static int rfkill_rk_remove(struct platform_device *pdev)
         gpio_free(rfkill->pdata->reset_gpio.io);
     
     if (gpio_is_valid(rfkill->pdata->poweron_gpio.io))
-        gpio_free(rfkill->pdata->poweron_gpio.io);
+		gpio_free(rfkill->pdata->poweron_gpio.io);
 	clk_disable_unprepare(rfkill->pdata->ext_clk);
     g_rfkill = NULL;
 
-- 
2.35.3

